### PR TITLE
Add validator extension

### PR DESCRIPTION
**Pull request by dslik**
First formal draft of the validator extension.

Need to fix several documentation build issues with the newer version of sphinx:
- code examples have extra blank lines, and no longer wrap across pages correctly - DONE
- table formatting is broken - DONE

For the validator extension, the following items need to be completed:
- Additional examples
- Verify that open questions are closed off.

### CODE REVIEW

**Comment by dslik** [comment]
Fixed tables and code examples. Cause was updated defaults, and a conflict between the lineno package and code-block.

**Comment by dslik** [comment]
Verified examples.

Built PDF included.

File: cdmi_extensions/validators/license.md
Comment (garymazz) [inline cdmi_extensions/validators/license.md:1]: Update copyright 2023 to 2024

**Review by garymazz — APPROVED** [review]
The proposal looks good. The are a few grammar adjustments and the copyright dates. These will be commented when it reaches ISO review.

File: cdmi_extensions/validators/index.txt [hunk 1]
```diff
@@ -0,0 +1,567 @@
+CDMI Extension
+==============
+
+Validators CDMI Extension
+*************************
+
+Overview
+--------
+
+Some CDMI systems allow validation to be performed against CDMI objects. In such a system, multiple validations may be performed simultaneously against the same or multiple objects. In these systems, the client receives an indication of validation failure on object creation or modification, with validation success either resulting in normal HTTP responses being returned, or an indication of validation failure annotated as object metadata.
```
[the remark this answers is not in the document]
Comment (dslik) [reply cdmi_extensions/validators/index.txt:10]: Applied to next draft.

File: cdmi_extensions/validators/license.md
Comment (dslik) [reply cdmi_extensions/validators/license.md:1]: Updated.

**Comment by dslik** [comment]
No objections from the TWG to move to public review.

**Pull request merged**